From: Ben Hutchings <ben@decadent.org.uk>
Date: Thu, 13 Nov 2014 21:50:47 +0000
Subject: ceph: Avoid ABI change in 3.2.64
Forwarded: not-needed

ceph_msg::front_max was renamed to ceph_msg::front_alloc_len, but has
the same purpose.  Hide this from genksyms.

--- a/include/linux/ceph/messenger.h
+++ b/include/linux/ceph/messenger.h
@@ -92,7 +92,11 @@ struct ceph_msg {
 	bool front_is_vmalloc;
 	bool more_to_follow;
 	bool needs_out_seq;
+#ifdef __GENKSYMS__
+	int front_max;			/* renamed */
+#else
 	int front_alloc_len;
+#endif
 	unsigned long ack_stamp;        /* tx: when we were acked */
 
 	struct ceph_msgpool *pool;
